src/style.css

```
@@ -1,3 +1,58 @@
+:root {
+  --accent-color: #00acdc;
+}
+
 body {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+ul {
+  list-style-type: none;
+}
```

Reset default ul padding and margin
```suggestion
ul {
  list-style-type: none;
  margin: 0;
  padding: 0;
}
```